meta-mender-tegra/README.md

```diff
@@ -28,3 +28,20 @@ Build
 
     $ bitbake core-image-base
 
+## Customizing Image Rootfs Size
+
+To customize the size reserved for your rootfs, add variable MENDER_STORAGE_TOTAL_SIZE_MB to your build/local.conf or image and define with a value which is twice the size of your desired rootfs in MB.
```

Comment:
I do not know if this is needed here, and should be covered in docs.mender.io pretty well.

Reply:
Good point.  It didn't work right/well before warrior on tegra.  Probably better in a mender hub known issues section.

Reply:
I've moved to the updated known issues section on [mender hub](https://hub.mender.io/t/nvidia-tegra-jetson-tx2/123)